howto/update/upgrade-anbox.md

```diff
@@ -6,13 +6,38 @@ The upgrade instructions detail the revisions each charm needs to be upgraded to
 
 [note type="caution" status="Warning"]Before you perform the upgrade ensure that you perform a backup of critical data you don't want to lose.[/note]
 
+## Before you begin
+
+As with all upgrades, there is a possibility that there may be unforeseen difficulties. It is highly recommended that you make a backup of any important data, including any running workloads.
+
+You should also make sure:
+
+* Your deployment is running normally
+* Your Juju client and controller/models are running the latest version
+* You read the release notes for the version you are upgrading to, which will alert you to any important changes to the operation of your cluster
+
+[note type="information" status="Note"]
+
+The following assume you're using Juju >= 3.1. If you're using Juju 2.9 you have to map the following commands:
```

Comment:
So... Which should they use? Just above, you wrote: 
> Your Juju client and controller/models are running the latest version

Shouldn't that ensure they are running the latest version?

Reply:
It's tricky and depends on various other things (e.g. 3.x doesn't support bionic but Landscape still requires bionic so cannot be deployed with anything other but 2.9. Then not everyone is using Landscape, ...). 

From the upgrade perspective we just have to account for different versions.